src/Sulu/Bundle/SecurityBundle/EventListener/SuluSecurityListener.php

```diff
@@ -40,12 +42,14 @@ public function onKernelController(FilterControllerEvent $event)
     {
         $controller = $event->getController();
 
-        if (!$controller[0] instanceof SecuredControllerInterface) {
+        if (
+            !$controller[0] instanceof SecuredControllerInterface &&
```

Comment:
We should probably assign `$conteoller[0]` to its own variable rather than use the index `0` all of the time